test/linreg/genData.py

```diff
@@ -0,0 +1,37 @@
+# Samuel Havron
+# https://www.github.com/samuelhavron
+# MIT License
```

Comment:
Although I will grant you that the MIT license is essentially identical to BSD, it probably makes sense to License all of Obliv-C the same way.